tests/CppUTestExt/IEEE754PluginTest.cpp

```diff
@@ -27,59 +27,67 @@
 
 /* The IEEE754ExceptionFlags plugin by definition requires C++11 */
 
-#ifdef CPPUTEST_COMPILER_FULLY_SUPPORTS_CXX11
-
 #include "CppUTest/CommandLineTestRunner.h"
 #include "CppUTest/TestHarness.h"
 #include "CppUTest/TestRegistry.h"
 #include "CppUTest/TestTestingFixture.h"
 #include "CppUTestExt/IEEE754ExceptionsPlugin.h"
-#include <cfenv>
 
 extern "C" { 
     #include "IEEE754PluginTest_c.h"
 }
 
+#if CPPUTEST_USE_STD_C_LIB
```

Comment:
I'd prefer not to have more of the IGNORE_TEST for this reason and keep it simpler. I guess there are better ways to discover code under #if that isn't used. Must be tools for that that we can integrate in the build. (as it just requires the C preprocessor)